src/include/pbs_ifl.h

```diff
@@ -315,6 +316,7 @@ extern "C" {
 #define ATTR_max_concurrent_prov "max_concurrent_provision"
 #define ATTR_resv_post_processing "resv_post_processing_time"
 #define ATTR_backfill_depth "backfill_depth"
+#define ATTR_clearesten "clear_estimates_enable"
```

Comment:
How about `auto_clear_topjob_estimates` ?
I hate naming.  Maybe just go with yours and add in topjob.  Make it 'clear_topjob_estimates_enable`
It's still not quite right because we're not clearing the estimates all the time.  We're only clearing them when they are no longer top jobs.

Reply:
Changed to `clear_topjob_estimates_enable`. I prefer this version because it is consistent with other bool attributes having the `_enable` suffix.